admin/kernel-syncdata-service/src/main/java/io/mosip/kernel/syncdata/repository/IndividualTypeRepository.java

```diff
@@ -0,0 +1,39 @@
+package io.mosip.kernel.syncdata.repository;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import io.mosip.kernel.syncdata.dto.EntityDtimes;
+import org.springframework.cache.annotation.Cacheable;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import io.mosip.kernel.syncdata.entity.IndividualType;
+import io.mosip.kernel.syncdata.entity.id.CodeAndLanguageCodeID;
+
+/**
+ * Repository class to handle CRUD operations for
+ * {@link IndividualTypeRepository}}
+ * 
+ * @author Srinivasan
```

Comment:
Kindly add a note here that this is deprecated, and its added only for backward compatibility

Reply:
Added